packages/antd/README.md

```diff
@@ -56,8 +56,8 @@ Ant Design theme, fields and widgets for `react-jsonschema-form`.
 
 ### Prerequisites
 
-- `antd >= 4.0.0`
-- `@ant-design/icons >= 4.0.0`
+- `antd >= 4.7.0`
+- `@ant-design/icons >= 4.7.0`
```

Comment:
@ianwith if you could make another PR that bumps the peer dependency of antd to 4.7, we can then merge that in once we release the next major version of rjsf.